homework/style.css

```diff
@@ -1,3 +1,186 @@
 .alert-error {
```

Comment:
Your CSS styling still leaves to be desired, but you can address that in the week 2 homework. But this is what I get when I start your application on my wide-screen monitor:

![screenshot 2018-12-14 at 11 40 52](https://user-images.githubusercontent.com/2788771/49998870-aea41900-ff95-11e8-8529-e9cedbe4b534.png)